src/stories/Blocks/header/header-sticky.js

```diff
@@ -0,0 +1,48 @@
+window.addEventListener("DOMContentLoaded", () => {
+  let scrollDirection = "up";
+  let lastScrollY = window.scrollY;
+  // Get the starting position of the header
+  const initialHeaderYposition = document.querySelector("header").offsetTop;
+  // Tweak the header position when it's down by this amount.
+  const headerDownTopPositionOffset = 5;
+  const waitPxBeforeScroll = 5;
+
+  // If we're at the top of the page, make sure the header class is "up".
+  if (window.scrollY === 0) {
+    document.querySelector("header").classList.remove("down");
+    document.querySelector("header").classList.add("up");
```

Comment:
You are right. They were used initially but since I needed to calculate the height of the header and used that in the style attribute they lost their purpose. I'll remove them...